_docs/development/environment.md

```diff
@@ -9,62 +9,81 @@ menu:
 
 ```
 /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
 ```
 
 ### Update your PATH variable
 
-Manifold relies on a number of binstubs. To simplify development, ensure that ./bin and ./node_modules/.bin are both in your $PATH.
+Manifold relies on a number of binstubs. To simplify development, ensure that `./bin` and `./node_modules/.bin` are both in your $PATH.
 
+On MacOS 10.15 (Catalina) the default terminal shell was changed form bash to zsh. **Zsh and bash are similar, but are not interchangable**. If you don't already know which shell you are using, this next command will tell you.
+
+```
+echo $0
+```
+
+If `$0` responded with "zsh" then we are going to save our zsh configuration file to a variable called "profile"
+
+```
+profile=~/.zshrc
 ```
-touch ~/.bash_profile
-echo 'export PATH=./bin:$PATH' >> ~/.bash_profile
-echo 'export PATH=./node_modules/.bin:$PATH' >> ~/.bash_profile
-source ~/.bash_profile
+
+If `$0` responded with "bash", then we are going to save our bash configuration file to a variable called "profile"
+
+```
+profile=~/.bash_profile
+```
+
+Now we will modify our path using that profile variable we created earlier
+
+```
+touch $profile
+echo 'export PATH=./bin:$PATH' >> $profile
+echo 'export PATH=./node_modules/.bin:$PATH' >> $profile
+source $profile
 ```
 
 ### Install rbenv and Ruby
 
 Rbenv is a tool for managing the available Ruby versions on your computer. If you prefer .rvm or another mechanism for managing ruby versions, by all means use it, as long as you can get the correct version of Ruby installed.
 
-As of November 2018, Manifold runs on Ruby 2.3.5. The version changes relatively frequently, as new versions of Ruby are released. We store the current [required ruby version](https://github.com/ManifoldScholar/manifold/blob/master/.ruby-version) in a `.ruby-version` file in the project root. Manifold should work with version 2.3 or higher.
+As of January 2020, Manifold runs on Ruby 2.6.3. The version changes relatively frequently, as new versions of Ruby are released. We store the current [required ruby version](https://github.com/ManifoldScholar/manifold/blob/master/.ruby-version) in a `.ruby-version` file in the project root. Manifold should work with version 2.6 or higher.
 
 First, install rbenv.
 
 ```
 brew install rbenv
 ```
 
-Second, adjust your [.bash_profile file](https://www.gnu.org/software/bash/manual/html_node/Bash-Startup-Files.html) so that rbenv is initialized when you open a new terminal session. If you don't use Bash, consult [rbenv's documentation](https://github.com/rbenv/rbenv#basic-github-checkout).
+Second, adjust your profile so that rbenv is initialized when you open a new terminal session. Follow the directions given from running:
 
 ```
-touch ~/.bash_profile
-echo 'eval "$(rbenv init -)"' >> ~/.bash_profile
-source ~/.bash_profile
+rbenv init
```

Comment:
If you run that command, it will respond with this for zsh users:

```
# Load rbenv automatically by appending
# the following to ~/.zshrc:

eval "$(rbenv init -)"
```
and this for bash users:


```
# Load rbenv automatically by appending
# the following to ~/.bash_profile:

eval "$(rbenv init -)"
```

Another option is to run:
```
echo $(rbenv init) 2> $profile
```
Which will dump the whole response (command and comments) in to their profile. I think that might annoy experienced devs to have those comments in their profile, but make it easier for newer devs. Any preferences?